.github/workflows/daily.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  daily:
+    runs-on: ubuntu-latest
+    if: contains(github.ref, 'refs/heads/master')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Generate top 100
+        run: go run . -count 100 > top-100.tsv
+
+      - name: Generate top 1000
+        run: go run . -count 1000 > top-1000.tsv
+
+      - name: Generate top 10000
+        run: go run . -count 10000 > top-10000.tsv
```

Comment:
also, these three steps are a bit wasteful. just generate the 10k list, then use `sed` to chunk off the top 1k and 100.

Reply:
Great idea!